### PR TITLE
DOCS-1254: Remove decommissioning page from Cloud docs

### PR DESCRIPTION
**Pull request by girishshankaran**
Product Version(s):

Issue:
(https://tigera.atlassian.net/browse/DOCS-1254)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by girishshankaran** [comment]
@tmjd @ctauchen Can you please review this?

**Comment by ctauchen** [comment]
LGTM

**Review by ctauchen — APPROVED** [review]
Thanks for the quick turnaround on this @girishshankaran

File: calico-cloud_versioned_docs/version-3.15/reference/component-resources/node/configuration.mdx [hunk 1]
```diff
@@ -300,8 +300,7 @@ Substitute `[flag]` with one or more of the following.
 - `-felix-ready`
 
 The BIRD readiness endpoint ensures that the BGP mesh is healthy by verifying that all BGP peers are established and
-no graceful restart is in progress. If the BIRD readiness check is failing due to unreachable peers that are no longer
```
Comment (tmjd) [inline calico-cloud_versioned_docs/version-3.15/reference/component-resources/node/configuration.mdx:303]: I think we shouldn't be making changes to versions we're no longer using or serving. In other words I don't think you should make this change to the CC 3.15 docs.

**Review by bmckercher123 — APPROVED** [review]
LGTM

**Pull request merged**